tools/grambase.tex

```diff
@@ -1,66 +0,0 @@
-\infannex{gram}{Grammar summary}
```

Comment:
Oh, I see, you kept this in the formerly-generated grammar.tex file. Never mind, then.